text/message_recovery.go

```diff
@@ -46,6 +55,24 @@ func NewErrorValidationRecoveryTokenInvalidOrAlreadyUsed() *Message {
 	}
 }
 
+func NewErrorValidationRecoveryCodeInvalidOrAlreadyUsed() *Message {
+	return &Message{
+		ID:      ErrorValidationRecoveryCodeInvalidOrAlreadyUsed,
+		Text:    "The recovery code is invalid or has already been used. Please try again.",
+		Type:    Error,
+		Context: context(nil),
+	}
+}
+
+func NewErrorValidationRecoveryFlowSubmittedTooOften() *Message {
+	return &Message{
+		ID:      ErrorValidationRecoveryCodeUsedTooOften,
+		Text:    "The recovery was submitted too often. Please restart the flow.",
```

Comment:
I think we are showing a new flow with a zero submit count and a new code, and can remove the text around "flow" which might be confusing to everyday internet users:

```
		Text:    "The recovery was submitted too often. Please try again.",
```

Reply:
Sure, sounds good.